resolwe_bio/processes/variant_calling/gatk_variant_refinement.py

```diff
@@ -0,0 +1,126 @@
+"""Run GATK genotype refinement."""
+
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkGenotypeRefinement(Process):
+    """Run GATK Genotype Refinement."""
+
+    slug = "gatk-refine-variants"
+    name = "GATK refine variants"
+    category = "GATK"
+    process_type = "data:variants:variants:refinevariants"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {"type": "sample"}
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.1.0"}
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+        },
+    }
+    data_name = '{{ vcf|sample_name|default("?") }}'
+
+    class Input:
+        """Input fields for GatkGenotypeRefinement."""
+
+        vcf_main = DataField(
```

Comment:
Use just `vcf` for this input, so that the variant file input name in this process is the same as in the existing  VQSR process.